cmd/juju/cloud/show_test.go

```diff
@@ -55,24 +85,26 @@ clouds:
         endpoint: http://london/1.0
     config:
       bootstrap-timeout: 1800
```

Comment:
bootstrap-timeout is a bootstrap only config attr so it should be under bootstrap-config right?
Also, we need tests where the cloud.yaml file defines generic model config in addtion to bootstrap config and cloud specific config